README.md

```diff
@@ -1,24 +1,174 @@
-# README
+<a name="readme-top"></a>
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+<div align="center">
 
-Things you may want to cover:
+  <h3><b>**Recipe App**</b></h3>
 
-* Ruby version
+</div>
 
-* System dependencies
+<!-- TABLE OF CONTENTS -->
 
-* Configuration
+# 📗 Table of Contents
 
-* Database creation
+- [📗 Table of Contents](#-table-of-contents)
+- [📖 \[Catalog of my things\] ](#--)
+  - [🛠 Built With ](#-built-with-)
+    - [Tech Stack ](#tech-stack-)
+    - [Key Features ](#key-features-)
+  - [💻 Getting Started ](#-getting-started-)
+    - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
+    - [Install](#install)
+  - [🔭 Future Features ](#-future-features-)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [📝 License ](#-license-)
 
-* Database initialization
+<!-- PROJECT DESCRIPTION -->
 
-* How to run the test suite
+# 📖 [Recipe App] <a name="about-project"></a>
 
-* Services (job queues, cache servers, search engines, etc.)
+**[Recipe APP]** keeps track of all your recipes, ingredients, and inventory. It will allow you to save ingredients, keep track of what you have, create recipes, and generate a shopping list based on what you have and what you are missing from a recipe. Also, since sharing recipes is an important part of cooking the app should allow you to make them public so anyone can access them.
 
-* Deployment instructions
+## 🛠 Built With <a name="built-with"></a>
 
-* ...
+### Tech Stack <a name="tech-stack"></a>
+
+
+<details>
+  <summary>Ruby</summary>
+  <summary>Rails</summary>
+  <summary>IRB</summary>
+  <summary>SQL</summary>
+  <summary>Rspec</summary>
+  <summary>Vscode</summary>
+  <summary>Devise</summary>
+  <summary>Capybara</summary>
+</details>
+
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+- **[Schema file is created for the project]**
+- **[Database is connected to rails app]**
+- **[Use database migration files to maintain database schema]**
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+
+To get a local copy up and running, follow these steps.
+
+
+### Prerequisites
+
+In order to run this project you need:
+
+
+### Setup
+Clone this repository to your desired folder:
+
+```sh
+  git clone https://github.com/adeola003/Recipe_App.git
+  cd Recipe_app
+```
```

Comment:
- [ ] As suggested by the previous reviewer KIndly update your commands as it would be better if you put App with capital `A` in the Recipe_app such as **Recipe_App** to make your commands useful in running the app on local machines :blush: